site/content/docs/latest/project/.placeholder

```diff
@@ -0,0 +1,11 @@
+This directory is expected to contain symlinks to certain files of the root project directory. This way, these files will be rendered in Hugo.
```

Comment:
As we can't add external (not in the hugo folder) files for them to be rendered as normal files, we need to create a symlink so that they get "virtually" into the website folder.